### DISCUSSION TITLE
Show Relation Managers on Resource Create Page

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#5200] [Help] [answered]
Is it possible to show the relation managers on the resource create page? The relation managers only show on the edit page currently, and I'd like them to appear on the create page as well.

**Comment by danharrin** [comment, marked as answer]
No, because the owner record has not been created yet. You'd need to use a Repeater with `relationship()`

**Reply by intrepidws** [reply]
Thanks Dan, helpful as always!
